src/Steam.package/STListComponent.class/instance/report.st

```diff
@@ -0,0 +1,21 @@
+rendering
+report
```

Comment:
Open issue, report is not updated when we login/logout or we create/remove/update objects in the list.